### PR TITLE
chore: update babel to test auto changeset addition

### PR DESCRIPTION
**Pull request by tdeekens**

### CODE REVIEW

File: .github/.renovaterc.json [hunk 1]
```diff
@@ -16,6 +16,18 @@
       "updateTypes": ["minor", "patch"],
       "groupName": "all dependencies",
       "groupSlug": "all"
+    },
+    {
+      "updateTypes": ["major"],
+      "labels": ["changeset:patch"]
```
Comment (emmenko) [inline .github/.renovaterc.json:22]: `changeset:major`

File: .github/.renovaterc.json [hunk 2]
```diff
@@ -16,6 +16,18 @@
...
+    },
+    {
+      "updateTypes": ["minor"],
+      "labels": ["changeset:patch"]
```
Comment (emmenko) [inline .github/.renovaterc.json:26]: `changeset:minor`

Comment (tdeekens) [reply .github/.renovaterc.json:26]: Yes. I am not super sure yet about it. Not every major dep bump to the lib is a major bump to consumers. 

Also I need to drop the usage of the action. It doesn't support mono repos.

Comment (emmenko) [reply .github/.renovaterc.json:26]: Ah ok, I thought it was a typo or something

**Pull request merged**